tool_list.yaml

```diff
@@ -3200,3 +3200,9 @@ tools:
 - name: omero_get_value
   owner: ufz
   tool_panel_section_label: Imaging
+- name: split_file_on_column
+  owner: bgruening
+  tool_panel_section_label: Text Manipulation
+- name: xls2tabular
```

Comment:
I'm fine with `split_file_on_column`.

With `xls2tabular` there are a few problems: 

- Most importantly there is no container yet: https://github.com/galaxyproject/planemo-monitor/pull/13
- Also the tool repo could need testing and the tool a few improvements (e.g. not accept `data`)

I will work on this. For the time being, I would suggest to create a separate PR for adding `split_file_on_column`.